README.md

```diff
@@ -1,22 +1,27 @@
-# nats-request-multi-js
+# nats-multi-request-js
```

Comment:
It strikes me that the name is a little misleading, since technically it's a single request that could have multiple responses. But I'm not sure what a better name would be.